src/main/webapp/views/tutorials/vendor.html

```diff
@@ -0,0 +1,29 @@
+<div class="carousel-item active">
+  <div class="container">
+    <div class="row align-items-center justify-content-center">
+      <div class="col-md-auto col-sm-auto">
+        <iframe width="500" height="315"
+          src="https://www.youtube.com/embed/9iPvLx7gotk?autoplay=1">
+        </iframe>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="carousel-item">
+  <div class="container">
+    <div class="row align-items-center justify-content-center">
+      <div class="col-xs-auto col-md-auto col-sm-auto">
+        <div style="text-align:center; font-size:5vw;">Placeholder 1</div>
```

Comment:
Could you transfer these styles to the dedicated `style.css` file ?

Reply:
Actually I think is for the best to keep the styles in this way since this div will be deleted in the future when the tutorial content is inserted, what do you think?